### PR TITLE
Added button Style to set text and border to white

### LINKED ISSUE #1668: Add back button to close Cookie Notice.
### Overview
We need the "Got It" button on the Cookies and Privacy Policy modal to be visible for users to interact with it.

### More Info
Style changes might have taken away our ability to see and close the Cookie Notice. We need to refer back to design and update.

### Action Items
- [ ] adjust styles for the "Got it" button to match the desired look (see pictures in Resources)

### Resources/Instructions
<details><summary>Current screenshot of transparent "Got it" button</summary>
<p>

image

</p>
</details> 

<details><summary>Screenshot of how "Got it" should look</summary>
<p>

image

</p>
</details>

### PR DESCRIPTION
**Pull request by funbunch**
Fixes #1668 

  - [ ] Up to date with `main` branch
  - [ ] Branch name follows [guidelines](https://github.com/hackforla/311-data/blob/master/GETTING_STARTED.md#feature-branching)
  - [ ] All PR Status checks are successful
  - [ ] Peer reviewed and approved

Any questions? See the [getting started guide](https://github.com/hackforla/311-data/blob/master/GETTING_STARTED.md)

### CODE REVIEW

**Comment by funbunch** [comment]
@ryanfchase Changed colors to be visible now. I wasn't sure what if we wanted a hover effect so I left that as is for now. 

See preview: 
Screenshot 2024-02-06 at 7 12 03 PM

**Review by sophia-bui — APPROVED** [review]
The styling looks great, Shannon!

File: components/main/CookieNotice.jsx [hunk 1]
```diff
@@ -76,7 +83,7 @@ function CookieNotice({
           website, you consent to our use of cookies and other tracking technologies.
           <Link className={classes.linkStyle} to="/privacy">Learn more</Link>
           <CardActions style={{ justifyContent: 'flex-end' }}>
-            <Button variant="outlined" onClick={handleClick}>Got it!</Button>
+            <Button className={classes.buttonStyle} variant="outlined" onClick={handleClick}>Got it!</Button>
```
Comment (Skydodle) [inline components/main/CookieNotice.jsx:86]: Although this does work visually, for best practice we should try to avoid hard coding extra styles while utilizing what MUI already provides. The Button component we are using is imported from MUI, therefore we can simply change the variant to 'contained':
`<Button variant="contained" onClick={handleClick}>Got it!</Button>`

<details><summary>The visual result of variant 'contained'</summary>

<p>

Screenshot 2024-02-08 at 1 06 03 PM


</p>
</details> 

Reference: [MUI React Button Component](https://mui.com/material-ui/react-button/)

**Review by Skydodle — CHANGES_REQUESTED** [review]
Please utilize the MUI Button component's variant instead.

File: components/main/CookieNotice.jsx (hunk 1 above)
Comment (funbunch) [reply components/main/CookieNotice.jsx:86]: Hi @Skydodle I did hope to use the built-in MUI styles. As you see the variant was already set to 'outlined' but 
for some reason wasn't appearing with an outline like in their docs. I did see 'contained' would work to change the look, but in our design, we wanted a white outline. Also, the color of the text without the styles I added is "color: #29404F;". I'm open to learning how we can do this with MUI.

Comment (Skydodle) [reply components/main/CookieNotice.jsx:86]: @funbunch Thank you for the clarification with the white outline requirement. I'm going to approve the PR and please kindly merge at your convenience.

With MUI there are 6 [default colors](https://mui.com/material-ui/customization/palette/#default-colors) profiles. The reason that 'outlined' variant color appears as it is right now, is because there is no 'color' property specified and it defaults to 'primary' color. We have a file that customizes 2 of 6 color profiles at `theme/colors.js`, in this file you will see the color '#29404F' you were talking about.

If you change the color profile to 'secondary' it will become #A8A8A8 which is textSecondaryDark from the color file, it's closest profile to what we want to do but still not white at all. Just want to bring this up for you to play around with.
`<Button variant="outlined" color="secondary" onClick={handleClick}>Got it!</Button>`
<details><summary>color profile options</summary>
<p>

Screenshot 2024-02-10 at 10 22 25 PM


</p>
</details> 

To [customize colors](https://mui.com/material-ui/customization/palette/#custom-colors) in a local file we are suppose to use the createTheme implementation for MUI 5 (we updated 5 few months ago), but we havent got around to update all the makeStyle  (from MUI4, deprecated) to createTheme, so your implementation is 100% correct.

Another workaround is to add the white you want by adding it to a color profile in `theme/colors.js`, maybe 'info' or 'warning' as a global color for future use (not sure if we'll need it). I'll leave you to decide.

In short, the MUI default color profiles are customizable globally for things that can be used throughout the app (the theme/colors file), and the customize color method are for local use (like the makeStyle or createTheme).

I hope this helps!

Comment (funbunch) [reply components/main/CookieNotice.jsx]: @Skydodle Super helpful explanation! That makes sense now. Updated this to use the "inherit' option. And I'll go ahead and merge. 
Thanks!
